From 3d9d56092db59a2604af3b59c1fe5b98c9a1bc3f Mon Sep 17 00:00:00 2001
From: Preeti Sachan <preetixx.sachan@intel.com>
Date: Tue, 13 Mar 2018 16:49:53 +0800
Subject: [PATCH] Fix wayland-protocols lookup

Prefix the path returned by pkg-config with PKG_CONFIG_SYSROOT_DIR.
Fix path for variable WAYLAND_PROTOCOLS_DATADIR.

Signed-off-by: Preeti Sachan <preetixx.sachan@intel.com>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index d809359..7ea7b46 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1780,7 +1780,7 @@ for plat in $platforms; do
         PKG_CHECK_MODULES([WAYLAND_CLIENT], [wayland-client >= $WAYLAND_REQUIRED])
         PKG_CHECK_MODULES([WAYLAND_SERVER], [wayland-server >= $WAYLAND_REQUIRED])
         PKG_CHECK_MODULES([WAYLAND_PROTOCOLS], [wayland-protocols >= $WAYLAND_PROTOCOLS_REQUIRED])
-        WAYLAND_PROTOCOLS_DATADIR=`$PKG_CONFIG --variable=pkgdatadir wayland-protocols`
+        WAYLAND_PROTOCOLS_DATADIR=$PKG_CONFIG_SYSROOT_DIR`$PKG_CONFIG --variable=pkgdatadir wayland-protocols`
 
         PKG_CHECK_MODULES([WAYLAND_SCANNER], [wayland-scanner],
                           WAYLAND_SCANNER=`$PKG_CONFIG --variable=wayland_scanner wayland-scanner`,
-- 
1.9.1

